spec/tapioca/cli_spec.rb

```diff
@@ -4,6 +4,7 @@
 require "spec_helper"
 require "pathname"
 require "shellwords"
+require "pry"
```

Comment:
debug statement

Reply:
Well.. TIL that `pry` does something with `$?` because when using that initially I was getting deprecation errors advising to use `$CHILD_STATUS`. Then after removing this line I get failing tests... turns out `$CHILD_STATUS` exists on `pry` so had to go back to `$?`...